docs/consensus/migration-to-pos.md

```diff
@@ -0,0 +1,20 @@
+---
+id: migration-to-pos
+title: Migration from PoA to PoS
+---
+
+## Overview
+
+This section guides you through migration from PoA to PoS for the running cluster without reset of blockchain.
+
+## How to migrate to PoS
+
+You will need to stop all nodes, update genesis.json, and restart the nodes in order to switch to PoS. `ibft switch` command append fork configuration into `genesis.json`.
```

Comment:
What's unclear to me - by reading this I get the impression that I need to stop the nodes, _manually edit the genesis.json_ and restart the nodes.

If I understood correctly, `ibft switch` does it for us while the cluster is running, it's just that we need to restart the nodes after we run `ibft switch`?

Reply:
yes

Reply:
Fixed at c54b7deb2c83464f83a348a110ea278cef3f0798